apps/web/src/pages/Skills/SkillShowcasePage.tsx

```diff
@@ -217,7 +217,7 @@ export const SkillShowcase = ({
                     description: "Description of a users top technical skills",
                   })}
                   editLink={{
-                    href: paths.skillShowcase(), // TO DO: Update with proper path in #7388
+                    href: paths.topTechnicalSkills(), // TO DO: Update with proper path in #7388
```

Comment:
Thanks for adding these, looks like the comment can be removed now 👍

Reply:
Oops, fixed by [6abfbfe](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7933/commits/6abfbfe693587a03f54f426e7c9a3c8d1ebc7b36)